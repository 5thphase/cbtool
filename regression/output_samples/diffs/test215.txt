|arrival |1401913336 
|command_originated |1401913334 
\|mgt_d \|X 
