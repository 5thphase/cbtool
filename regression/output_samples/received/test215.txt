The VM object vm_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|abort                                   |False                                
|access                                  |                                     
|ai                                      |none                                 
|ai_arrived                              |0                                    
|ai_arriving                             |0                                    
|ai_departed                             |0                                    
|ai_failed                               |0                                    
|ai_issued                               |0                                    
|ai_name                                 |none                                 
|ai_reservations                         |0                                    
|aidrs                                   |none                                 
|aidrs_name                              |none                                 
|alternative_remote_mtu                  |False                                
|alternative_remote_mtu_default          |1200                                 
|alternative_remote_mtu_interface        |default                              
|arrival                                 |1471275403                           
|base_dir                                |/home/msilva/cloudbench/lib/auxiliary//../..
|capture_supported                       |True                                 
|check_boot_complete                     |wait_for_0                           
|check_boot_started                      |poll_cloud                           
|class                                   |standard                             
|client_pref_pool                        |lg                                   
|cloud_hostname                          |71DEFC18-699B-5DCD-98DB-819FC4863605.simcloud.com
|cloud_init_bootstrap                    |False                                
|cloud_init_rsync                        |False                                
|cloud_ip                                |90.17.211.77                         
|cloud_mac                               |00:16:3e:06:c3:90                    
|cloud_name                              |TESTCLOUD                            
|cloud_vm_name                           |cb-msilva-vm-1-tinyvm                
|cloud_vm_uuid                           |71DEFC18-699B-5DCD-98DB-819FC4863605 
|command                                 |vmattach TESTCLOUD tinyvm auto empty default none empty=empty
|command_originated                      |1471275400                           
|comments                                |                                     
|counter                                 |209                                  
|cpu_lower                               |500                                  
|cpu_upper                               |1000                                 
|create_jumphost                         |False                                
|credentials                             |                                     
|credentials_dir                         |/home/msilva/cloudbench/lib/auxiliary//../../credentials
|daemon_dir                              |/home/klabuser                       
|debug_remote_commands                   |False                                
|detach_parallelism                      |20                                   
|disable_timesync                        |False                                
|discover_hosts                          |True                                 
|driver_pref_pool                        |lg                                   
|eclipsed                                |False                                
|errors                                  |no                                   
|exclude_list                            |/home/msilva/cloudbench/lib/auxiliary//../../exclude_list.txt
|execute_json_filename_prefix            |cb                                   
|execute_script_name                     |~/cloudbench/scenarios/scripts/openstack_multitenant.sh
|expected_mtu                            |1500                                 
|experiment_id                           |REGRESSION_B                         
|extra_info                              |                                     
|filestore_hostname                      |9.2.208.21                           
|filestore_port                          |10000                                
|filestore_username                      |msilva                               
|force_failure                           |False                                
|host                                    |B728D4A5-423F-512A-AB26-A792B1968DBC 
|host_cloud_ip                           |89.191.217.200                       
|host_name                               |simhostd3                            
|identity                                |/home/msilva/cloudbench/lib/auxiliary//../../credentials/cbtool_rsa
|imageid1                                |cb_nullworkload                      
|imageids                                |1                                    
|is_jumphost                             |False                                
|jars_dir                                |/home/klabuser/cloudbench/jar        
|jumphost_base_name                      |cb-jumphost                          
|jumphost_login                          |auto                                 
|jumphost_netnames                       |all                                  
|key_name                                |                                     
|last_known_state                        |generic post-boot script executed    
|leave_instance_on_failure               |False                                
|local_dir_name                          |cloudbench                           
|login                                   |klabuser                             
|memory_lower                            |50                                   
|memory_upper                            |100                                  
|meta_tags                               |empty                                
|mgt_001_provisioning_request_originated |1471275400                           
|mgt_002_provisioning_request_sent       |1                                    
|mgt_003_provisioning_request_completed  |1                                    
|mgt_004_network_acessible               |1                                    
|mgt_005_file_transfer                   |0                                    
|mgt_006_instance_preparation            |1                                    
|mgt_007_application_start               |0                                    
|migrate_protocol                        |tcp                                  
|migrate_protocol_supported              |tcp,rdma                             
|migrate_supported                       |True                                 
|mode                                    |controllable                         
|model                                   |sim                                  
|name                                    |vm_1                                 
|netname                                 |private                              
|notification                            |False                                
|notification_channel                    |auto                                 
|pattern                                 |none                                 
|pct_failure                             |                                     
|placement                               |random                               
|pre_creation_delay                      |0                                    
|protect_protocol                        |tcp                                  
|protect_protocol_supported              |tcp,rdma                             
|protect_supported                       |True                                 
|prov_cloud_ip                           |153.55.10.118                        
|prov_netname                            |private                              
|qemu_debug                              |False                                
|randomize_image_name                    |False                                
|recreate_attempts                       |1                                    
|remote_dir_name                         |cloudbench                           
|resize_supported                        |False                                
|role                                    |tinyvm                               
|run_cloud_ip                            |90.17.211.77                         
|run_generic_scripts                     |False                                
|run_netname                             |private                              
|runstate_parallelism                    |5                                    
|runstate_supported                      |True                                 
|security_groups                         |                                     
|sim_0021_vm_placement_time              |0                                    
|size                                    |pico32                               
|sla_provisioning_abort                  |False                                
|sla_runtime                             |ok                                   
|ssh_key_name                            |cbtool_rsa                           
|staging                                 |none                                 
|state                                   |attached                             
|sticky_app_status                       |False                                
|temp_attr_list                          |empty=empty                          
|tenant                                  |default                              
|timeout                                 |240                                  
|tracking                                |none                                 
|transfer_files                          |False                                
|type                                    |none                                 
|update_attempts                         |720                                  
|update_frequency                        |1                                    
|use_floating_ip                         |False                                
|use_jumphost                            |False                                
|use_vpn_ip                              |False                                
|userdata                                |/var/lib/cloud/71DEFC18-699B-5DCD-98DB-819FC4863605/user-data.txt
|username                                |msilva                               
|utc_offset_delta                        |-18000                               
|utc_offset_on_vm                        |3600                                 
|uuid                                    |4BB20172-486A-5DCB-9DC1-A49A5B8AFE4A 
|vcpus                                   |1                                    
|vm_arrived                              |0                                    
|vm_arriving                             |0                                    
|vm_departed                             |0                                    
|vm_failed                               |0                                    
|vm_issued                               |1                                    
|vm_reservations                         |0                                    
|vmc                                     |48E847B7-4EAD-595E-A603-912566454DDC 
|vmc_access                              |                                     
|vmc_arrived                             |16                                   
|vmc_cloud_ip                            |40.45.97.29                          
|vmc_departed                            |12                                   
|vmc_failed                              |0                                    
|vmc_issued                              |16                                   
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  
|vmemory                                 |192                                  
|vnics                                   |1                                    
|vpn_config_file                         |/home/msilva/cloudbench/lib/auxiliary//../../configs/generated/TESTCLOUD_client-cb-openvpn.conf
|vpn_only                                |False                                
|vpn_server_bootstrap                    |192.168.0.1                          
|vpn_server_ip                           |9.2.208.21                           
|vpn_server_port                         |1194                                 
|vstorage                                |2048                                 

