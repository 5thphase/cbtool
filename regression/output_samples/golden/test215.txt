The VM object vm_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|access                                  |                                     
|ai                                      |none                                 
|ai_arrived                              |0                                    
|ai_arriving                             |0                                    
|ai_departed                             |0                                    
|ai_failed                               |0                                    
|ai_issued                               |0                                    
|ai_name                                 |none                                 
|ai_reservations                         |0                                    
|aidrs                                   |none                                 
|aidrs_name                              |none                                 
|alternative_remote_mtu                  |False                                
|alternative_remote_mtu_default          |1200                                 
|alternative_remote_mtu_interface        |default                              
|arrival                                 |1454603929                           
|attempts                                |24                                   
|base_dir                                |/home/msilva/cloudbench/lib/auxiliary//../..
|capture_supported                       |True                                 
|check_boot_complete                     |wait_for_0                           
|check_boot_started                      |poll_cloud                           
|class                                   |standard                             
|client_pref_pool                        |lg                                   
|cloud_hostname                          |B54C0BA7-3E44-5E01-9E1E-E84C603C0B60.simcloud.com
|cloud_init_bootstrap                    |False                                
|cloud_init_rsync                        |False                                
|cloud_ip                                |4.81.141.120                         
|cloud_mac                               |00:16:3e:7d:30:a8                    
|cloud_name                              |TESTCLOUD                            
|cloud_vm_name                           |cb-msilva-vm-1-tinyvm                
|cloud_vm_uuid                           |B54C0BA7-3E44-5E01-9E1E-E84C603C0B60 
|command                                 |vmattach TESTCLOUD tinyvm auto empty default none empty=empty
|command_originated                      |1454603926                           
|comments                                |                                     
|counter                                 |209                                  
|cpu_lower                               |500                                  
|cpu_upper                               |1000                                 
|create_jumphost                         |False                                
|credentials                             |                                     
|credentials_dir                         |/home/msilva/cloudbench/lib/auxiliary//../../credentials
|daemon_dir                              |/home/klabuser                       
|debug_remote_commands                   |False                                
|detach_parallelism                      |20                                   
|discover_hosts                          |True                                 
|driver_pref_pool                        |lg                                   
|eclipsed                                |False                                
|errors                                  |no                                   
|exclude_list                            |/home/msilva/cloudbench/lib/auxiliary//../../exclude_list.txt
|execute_json_filename_prefix            |cb                                   
|execute_script_name                     |execute_on_staging.sh                
|expected_mtu                            |1500                                 
|experiment_id                           |REGRESSION_B                         
|extra_info                              |                                     
|filestore_hostname                      |172.16.0.4                           
|filestore_port                          |10000                                
|filestore_username                      |msilva                               
|force_failure                           |False                                
|host                                    |7F1EEEF6-28F4-5A63-80D8-15DF2092C1EC 
|host_cloud_ip                           |46.137.141.45                        
|host_name                               |simhostb0                            
|identity                                |/home/msilva/cloudbench/lib/auxiliary//../../credentials/cbtool_rsa
|imageid1                                |cb_nullworkload                      
|imageids                                |1                                    
|is_jumphost                             |False                                
|jars_dir                                |/home/klabuser/cloudbench/jar        
|jumphost_base_name                      |cb-jumphost                          
|jumphost_login                          |auto                                 
|jumphost_netnames                       |all                                  
|key_name                                |                                     
|last_known_state                        |running with ip assigned             
|leave_instance_on_failure               |False                                
|local_dir_name                          |cloudbench                           
|login                                   |klabuser                             
|memory_lower                            |50                                   
|memory_upper                            |100                                  
|meta_tags                               |empty                                
|mgt_001_provisioning_request_originated |1454603926                           
|mgt_002_provisioning_request_sent       |1                                    
|mgt_003_provisioning_request_completed  |1                                    
|mgt_004_network_acessible               |1                                    
|mgt_005_file_transfer                   |0                                    
|mgt_006_instance_preparation            |1                                    
|mgt_007_application_start               |0                                    
|migrate_protocol                        |tcp                                  
|migrate_protocol_supported              |tcp,rdma                             
|migrate_supported                       |True                                 
|mode                                    |controllable                         
|model                                   |sim                                  
|name                                    |vm_1                                 
|netname                                 |private                              
|notification                            |False                                
|notification_channel                    |auto                                 
|pattern                                 |none                                 
|placement                               |random                               
|protect_protocol                        |tcp                                  
|protect_protocol_supported              |tcp,rdma                             
|protect_supported                       |True                                 
|prov_cloud_ip                           |117.179.247.103                      
|prov_netname                            |private                              
|qemu_debug                              |False                                
|randomize_image_name                    |False                                
|remote_dir_name                         |cloudbench                           
|resize_supported                        |False                                
|role                                    |tinyvm                               
|run_cloud_ip                            |4.81.141.120                         
|run_generic_scripts                     |False                                
|run_netname                             |private                              
|runstate_parallelism                    |5                                    
|runstate_supported                      |True                                 
|security_groups                         |                                     
|sim_0021_vm_placement_time              |0                                    
|size                                    |pico32                               
|sla_runtime                             |ok                                   
|ssh_key_name                            |cbtool_rsa                           
|staging                                 |none                                 
|state                                   |attached                             
|sticky_app_status                       |False                                
|temp_attr_list                          |empty=empty                          
|tenant                                  |default                              
|timeout                                 |240                                  
|tracking                                |none                                 
|transfer_files                          |False                                
|type                                    |none                                 
|update_attempts                         |720                                  
|update_frequency                        |1                                    
|use_floating_ip                         |False                                
|use_jumphost                            |False                                
|use_vpn_ip                              |False                                
|userdata                                |/var/lib/cloud/B54C0BA7-3E44-5E01-9E1E-E84C603C0B60/user-data.txt
|username                                |msilva                               
|utc_offset_delta                        |-21600                               
|utc_offset_on_orchestrator              |-18000                               
|utc_offset_on_vm                        |3600                                 
|uuid                                    |A04D80E7-0BB2-55AD-A873-E01F14DD1C2A 
|vcpus                                   |1                                    
|vm_arrived                              |0                                    
|vm_arriving                             |0                                    
|vm_departed                             |0                                    
|vm_failed                               |0                                    
|vm_issued                               |1                                    
|vm_reservations                         |0                                    
|vmc                                     |FB2A884A-73A4-5C91-B913-9AD4AF1FB28A 
|vmc_access                              |                                     
|vmc_arrived                             |16                                   
|vmc_cloud_ip                            |148.69.140.12                        
|vmc_departed                            |12                                   
|vmc_failed                              |0                                    
|vmc_issued                              |16                                   
|vmc_name                                |simzone_c                            
|vmc_pool                                |SUT                                  
|vmemory                                 |192                                  
|vnics                                   |1                                    
|vpn_only                                |False                                
|vpn_server_bootstrap                    |192.168.0.6                          
|vpn_server_ip                           |9.2.208.228                          
|vpn_server_port                         |1194                                 
|vstorage                                |2048                                 

